automation/check-patch.e2e.packages

```diff
@@ -1,3 +1,4 @@
 docker
 git
 make
+shyaml
```

Comment:
why ?
at the end Ram used containerized version for it, can you do the same please if possible?

Reply:
Can you refere me to that code/script?
FYI, all scripts under automation are scripts that run on prow

Reply:
https://github.com/kubevirt/cluster-network-addons-operator/blob/42c44712947d5841ec4623279e84ad31cf27da57/hack/components/yaml-utils.sh#L5

doesnt matter where they run, this one will be generic

Reply:
Prow does not use this file, this is for the old stdci CI system.

Reply:
@qinqon @oshoval 
Got it, I will remove this change